doc/guides/writing-and-running-benchmarks.md

```diff
@@ -163,6 +163,20 @@ First build two versions of Node.js, one from the master branch (here called
 `./node-master`) and another with the pull request applied (here called
 `./node-pr-5134`).
 
+*To run multiple compiled versions in parallel you need to copy the output of the
```

Comment:
Could this not be italics? I think it flows well with the rest of the copy here so it doesn't need to stand out.